gemini/use-cases/retrieval-augmented-generation/developer_code_chat/developer_code_chat.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<p>Use code font for code sample</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>